source/_patterns/00-atoms/block/paragraph/paragraph.twig

```diff
@@ -1,7 +1,5 @@
-{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['paragraph']) }) -%}
-
-<p {%- include 'atoms-attributes' -%}>
+{%- import 'atoms-html' as html -%}
 
-  {%- include 'atoms-text' with {nodes: text} -%}
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['paragraph']) }) -%}
```

Comment:
Been going round the houses trying to find a way to improve this. Can't, and keep coming back to #40...